chat-plugins/info.js

```diff
@@ -14,7 +14,7 @@
 
 const RESULTS_MAX_LENGTH = 10;
 
-let commands = exports.commands = {
+let commands = exports.commands = { // eslint-disable-line no-unused-vars
```

Comment:
Nah, you can use `CommandParser.baseCommands` if you ever need it for `/eval`. That's what I always used, anyway; never realized it could be anything else.